app/src/main/java/fr/free/nrw/commons/upload/worker/UploadWorker.kt

```diff
@@ -469,18 +469,21 @@ class UploadWorker(
         contribution: Contribution,
     ) {
         val wikiDataPlace = contribution.wikidataPlace
-        if (wikiDataPlace != null && wikiDataPlace.imageValue == null) {
+
+        if (wikiDataPlace != null) {
             if (!contribution.hasInvalidLocation()) {
                 var revisionID: Long? = null
                 try {
-                    revisionID =
-                        wikidataEditService.createClaim(
-                            wikiDataPlace,
-                            uploadResult.filename,
-                            contribution.media.captions,
-                        )
-                    if (null != revisionID) {
+                    // Always attempt to update the image claim with the latest photo
+                    revisionID = wikidataEditService.createClaim(
```

Comment:
Please perform such stylistic changes in a separate style-only pull request. Thanks!

Reply:
Please revert this stylistic change (moving `wikidataEditService.createClaim(` to the same line, and indentation afterwards)